### ISSUE TITLE
Need to declare dependency upon core snap

### ISSUE THREAD

**Issue by tddpirate** [Lukewh/deno-snap#1]
TL;DR: declare dependency upon `core` snap rather than upon `core18` snap.

When installing `deno` using `snap`, it pulls in also the `core18` snap.
Then `deno run ~/tmp/dentest.js` fails with
`error: Cannot resolve module "file:///home/omer/tmp/dentest.ts"`

When I run the script in a `/snap/deno` subdirectory, it fails with
`error: Uncaught PermissionDenied: network access to "0.0.0.0:8000", run again with the --allow-net flag`
and I was not successful in getting rid of the error no matter what argument (or lack of argument) I gave to `--allow-net` flag.

Both problems disappeared when I manually installed the `core` snap.
```
$ snap connections deno
Interface     Plug               Slot           Notes
home          deno:home          :home          -
network       deno:network       :network       -
network-bind  deno:network-bind  :network-bind  -```

So it seems that `deno` needs to depend upon `core` rather than upon `core18`.
